projects/step-frontend/src/lib/modules/timeseries/performance-view/performance-view.component.ts

```diff
@@ -499,7 +500,10 @@ export class PerformanceViewComponent implements OnInit, OnDestroy {
     this.throughputChart.setData(data, false);
   }
 
-  getSeriesKey(attributes: BucketAttributes, groupDimensions: string[]) {
+  getSeriesKey(attributes: BucketAttributes, groupDimensions: string[]): string {
+    if (Object.keys(attributes).length === 0) {
```

Comment:
I know this ```.length === 0``` is considered more explicit, but JavaScript specifically gives us the ability to work with truthy and falsy values in order to simplify the code, why not just write ```!Object.keys(attributes).length```